content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,183 @@
+
+We are working on developing experiments to have features or documented patterns
+explicitly for grid search support, so definitely share any feedback to help
+drive the future direction of that!
+
+### [How can I pass sensitive strings (connectionString, containerName, etc) to DVC get without storing them to Git?](https://discord.com/channels/485586884165107732/563406153334128681/830021022337073185)
+
+There's a bit of context behind this question that might give it more meaning.
+
+---
+
+I set up a private GitHub repo to be a data registry and I have set up a private
+Azure remote where I have pushed some datasets.
+
+At the moment I set up the Azure remote using the environment variables
+[as the docs explain here](https://dvc.org/doc/command-reference/remote/add) so
+I don't push sensitive information.
+
+I am now trying to read those datasets from another repository
+("my-project-repo"), using `dvc get` (e.g.
+`dvc get git@github.com:data-registry-repo.git path/data.csv`) but I get this
+error:
+
+```bash
+ERROR: failed to get 'path/data.csv' from 'git@github.com:data-registry-repo.git' - Authentication to Azure Blob Storage via default credentials (https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.DefaultAzureCredential) failed.
+Learn more about configuration settings at <https://man.dvc.org/remote/modify>: unable to connect to account for Must provide either a connection_string or account_name with credentials!!
+```
+
+---
+
+Generally, there are two ways solve this issue:
+
+- ENV vars
+- setup some options in the `--global, --system` DVC config
+
+Once they removed some old remotes (with the same name) from their project repo
+and tried to pull from the data registry using `AZURE_STORAGE_CONNECTION_STRING`
+and `AZURE_STORAGE_CONTAINER_NAME`, it worked!
+
+### [Is there any way to ensure that `dvc import` uses the cache from the config file?](https://discord.com/channels/485586884165107732/563406153334128681/827574712825413672)
+
+This is another great question where a little context might be useful.
+
+---
+
+I'm trying to import a dataset project into another DVC project, let's call it
+_dvcdata_.
+
+The config for _dvcdata_ is:
+
+```bash
+[core]
+    remote = awsremote
+[cache]
+    type = symlink
+    dir = /home/user/dvc_cache
+['remote "awsremote"']
+    url = s3://...
+```
+
+When I run `dvc import git@github.com:user/dvcdata.git my_data`, it starts to
+redownload it. I have double checked that I have pushed this config file to
+master.
+
+I have also checked that if I manually clone the repo and run
+`cd /tmp/dvcdata; dvc pull`, the files are symlinked.
+
+The command output says it's downloading to
+`../../../../../../../tmp/tmphlgqpv9zdvc-clone/<file name>` so the cache isn't
+being updated.
+
+---
+
+The repo you are importing into has its own cache directory.
+
+If you want to use the same cache directory across both projects, you have to
```

Comment:
Can we recommend to use `--global` or `--system` (and do not write path to cache dir in the `.dvc/config` for neither of the project) for all projects and all machines?